code/modules/cargo/blackmarket/blackmarket_items/weapons.dm

```diff
@@ -304,6 +304,40 @@
 	stock = 2
 	availability_prob = 30
 
+/datum/blackmarket_item/weapon/mauler
+	name = "Mauler Machine Pistol"
+	desc = "This gun's got teeth! A Shoal classic, the Mauler 9mm machine pistol is suited to close quarter combat with an insane fire rate beating out even dedicated assault rifles."
```

Comment:
I'd mention the Mauler's small magazine capacity somewhere in here. It's a pretty defining trait of the gun so it feels like a notable omission.